lib/lexicon.py

```diff
@@ -104,6 +107,42 @@ def from_element(cls, e):
         synt = None if not synt else synt[0]
         return Lemma(orth, phon, synt, eval, special)
 
+    def _equals(self, other: Lemma, same_order: bool = True) -> bool:
```

Comment:
Why is the default `same_order=True` here, but the only usage of this function actually uses `same_order=False`?

The default should reflect the most reasonable expected common usage, or if this might be ambiguous, there should not be a default.

Reply:
You're right, probably the most common usage of equality would be checking without explicit order. At least I think it that way. What would other reviewers expect from the default of this function?